Home.html

```diff
@@ -14,64 +14,98 @@
-            </div>
-                <div class="main-program-divs d-flex ps-3 pe-3 ms-3 me-3 mb-2">
-                        <img src="assets/visa.png" alt="Visa">
-                        <h6>Visa</h6>
-                        <p>Visa application services for more than 10 countries</p>
-                </div>
-                <div class="main-program-divs d-flex ps-3 pe-3 ms-3 me-3 mb-2">
-                    <img src="assets/tickets.png" alt="Tickets">
-                    <h6>Tickets</h6>
-                    <p>We take care of the tickets so you don't have to worry about a thing</p>
-                </div>
-                <div class="main-program-divs d-flex ps-3 pe-3 ms-3 me-3 mb-2">
-                    <img src="assets/bank.png" alt="Bank">
-                    <h6>Bank Statement</h6>
-                    <p>Don't have a bank statement, no problem we will take care of that</p>
-                </div>
-                <div class="main-program-divs d-flex ps-3 pe-3 ms-3 me-3 mb-2">
-                    <img src="assets/patient.png" alt="passport">
-                    <h6>Medical Visa</h6>
-                    <p>Including all the medical files you need to apply for a medical visa</p>
+                <div class="program-container">
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/passport.png" alt="passport">
+                        <h6>passport</h6>
+                        <p>Online form application services for getting your passport in no time</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                            <img src="assets/visa.png" alt="Visa">
+                            <h6>Visa</h6>
+                            <p>Visa application services for more than 10 countries</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/tickets.png" alt="Tickets">
+                        <h6>Tickets</h6>
+                        <p>We take care of the tickets so you don't have to worry about a thing</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/bank.png" alt="Bank">
+                        <h6>Bank Statement</h6>
+                        <p>Don't have a bank statement, no problem we will take care of that</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/patient.png" alt="passport">
+                        <h6>Medical Visa</h6>
+                        <p>Including all the medical files you need to apply for a medical visa</p>
+                    </div>
```

Comment:
- [ ] Kindly make sure the `main-program` cards correctly match the Behance template design. All elements in each card should be correctly aligned, text should have correct size, height,...

### Template Design
![Screenshot from 2023-02-11 00-12-52](https://user-images.githubusercontent.com/83042742/218209295-c74de692-4a5d-4022-af16-3eb49d7340e8.png)

### Your Design 
![Screenshot from 2023-02-11 00-12-27](https://user-images.githubusercontent.com/83042742/218209347-75f2eeff-a777-445e-9896-884ea6921bcd.png)